docs/source/cn/array_ops.py

```diff
@@ -732,3 +732,197 @@
 
     """,
 )
+
+reset_docstr(
+    oneflow.load,
+    r"""加载一个被 oneflow.save() 保存的对象。
+
+    参数：
+        - **path** (str): 对象所在的路径
+        - **consistent_src_rank** (int, optional): 加载相容 (consistent) 张量的需要的秩 (rank)。被指定时，只有秩与 consistent_src_rank相等的进程才会真正读取 `path` 中的文件，并且被加载对象中的张量会与  placement = `flow.placement('cuda', [consistent_src_rank])` 一致。
+
+    返回类型：
+        加载好的对象
+    """
+)
+
+reset_docstr(
+    oneflow.numel,
+    r"""
+    numel(input) -> int
+
+    返回 :attr:`input` 张量中的元素总数量。
+
+    参数：
+        - **input** (oneflow.Tensor): 输入张量
+
+    .. code-block:: python
+
+        >>> import oneflow as flow
+
+        >>> a = flow.randn(1, 2, 3, 4, 5)
+        >>> flow.numel(a)
+        120
+        >>> a = flow.zeros(4,4)
+        >>> flow.numel(a)
+        16
+    """
+)
+
+reset_docstr(
+    oneflow.gather_nd,
+    r"""
+    oneflow.gather_nd(input, index) -> Tensor
+    
+    该算子为 `gather` 的高维扩展， `index` 是一个k维张量，以作为输入张量 `input` 的索引。
+    每个元素将定义一个 `input` 的切片：
+
+    .. math::
+
+        output[i_{0},i_{1},...,i_{K-2}] = input[index(i_{0},i_{1},...,i_{K-2})]
+
+
+    参数：
+        - **input**: 输入张量
+        - **index**: 切片索引
+
+    示例：
+
+    .. code-block:: python
+
+        >>> import oneflow as flow
+        >>> input = flow.tensor([[1, 2,3], [4, 5,6],[7,8,9]], dtype=flow.float)
+        >>> index_1 = flow.tensor([[0], [2]], dtype=flow.int)
+        >>> out_1 = flow.gather_nd(input,index_1)
+        >>> print(out_1.shape)
+        oneflow.Size([2, 3])
+        >>> out_1
+        tensor([[1., 2., 3.],
+                [7., 8., 9.]], dtype=oneflow.float32)
+        >>> index_2 = flow.tensor([[0,2], [2,1]], dtype=flow.int)
+        >>> out_2 = flow.gather_nd(input,index_2)
+        >>> out_2
+        tensor([3., 8.], dtype=oneflow.float32)
+
+    """,
+)
+
+reset_docstr(
+    oneflow.grad_enable,
+    r"""
+    启用梯度计算的上下文管理。
+
+    启用梯度计算，如果其被 no_grad 禁用。
+
+    这个上下文管理在本地线程上；不会影响其他线程的计算。
+
+    同时可以作为一种修饰模式。（请确保使用括号实例化）
+
+    .. code-block:: python
+
+        >>> import oneflow as flow
+        >>> x = flow.ones(2, 3, requires_grad=True)
+        >>> with flow.no_grad():
+        ...     with flow.grad_enable():
+        ...         y = x * x
+        >>> y.requires_grad
+        True
+        >>> @flow.grad_enable()
+        ... def no_grad_func(x):
+        ...     return x * x
+        >>> with flow.no_grad():
+        ...     y = no_grad_func(x)
+        >>> y.requires_grad
+        True
+    """
+)
+
+reset_docstr(
+    oneflow.inference_mode,
+    r"""
+    用于启用或禁用 inference mode 的上下文管理。
+
+    Inference mode 是一个新的上下文管理，类似于 no_grad ，可以在你确定你的运算将与 autograd 没有关联时被使用。在此模式下运行的代码将获得更好的性能，因为禁用了路径追踪和版本计数堆。
```

Comment:
```suggestion
    Inference mode 是一个新的上下文管理模式，类似于 no_grad ，当你确定你的运算与 autograd 没有关联时，可以使用这个模式。因为禁用了路径追踪和版本计数堆，在此模式下运行的代码将获得更好的性能。
```

也是翻译腔太重了，其它地方的翻译腔我暂时不看了，请重新自己检查调整下吧。

Reply:
已解决